mypy/subtypes.py

```diff
@@ -1,4 +1,4 @@
-from typing import List, Dict, Callable
+from typing import List, Dict, Callable, Optional
```

Comment:
You're right, they must have snuck in from another part of the stacked diffs I'm managing.